server/mldb_server.cc

```diff
@@ -263,6 +271,32 @@ runHttpQuery(const Utf8String& query,
                        rowNames, rowHashes, sortColumns);
 }
 
+void
+MldbServer::
+handleRedirectToGet(const string & uri,
+                    RestConnection & connection,
+                    const Json::Value & body) const
+{
```

Comment:
It's not possible.  We don't actually make an HTTP call; we simply call the handleRequest method.  So there is no security issue here.